### PR TITLE
Improve README.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

File: README-dev.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# Toit Package Manager - Development
+
+## Testing the image
+
+* Create a fresh repository or fork the existing registry repository.
+* Create a new ssh-key:
+  ``` shell
+  ssh-keygen -t ed25519
+  ```
+* Copy the public key into the deploy keys on Github: https://github.com/XXX/YYY/settings/keys.
```
Comment (kasperl) [inline README-dev.md:10]:
```suggestion
* Copy the public key into the deploy keys on GitHub: https://github.com/XXX/YYY/settings/keys.
```

File: README-dev.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+# Toit Package Manager - Development
```
Comment (kasperl) [inline README-dev.md:1]:
```suggestion
# Toit package manager - development
```

File: README-dev.md [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+  ```
+* Start docker if not already running `sudo systemctl start docker`
+* Run `make image`. Make sure `$HOME/go/bin` is in your path.
+* Start the image (see the main README):
+  ``` shell
+  docker run -p 8733:8733 -e"REGISTRY_URL=github.com/XXX/YYY" -e"..." toit_registry
+  ```
+
+
+## Tips and Tricks
```
Comment (kasperl) [inline README-dev.md:24]:
```suggestion
## Tips and tricks
```

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: README-dev.md (hunk 2 above)
Comment (floitsch) [reply README-dev.md]: done.

File: README-dev.md (hunk 1 above)
Comment (floitsch) [reply README-dev.md]: done.

File: README-dev.md (hunk 3 above)
Comment (floitsch) [reply README-dev.md]: done.

**Pull request merged**